examples/js/hello-world/app.js

```diff
@@ -115,23 +115,26 @@ app.post('/send/submit', (req, res) => {
     };
 
     // Get the DM channel between the user and the bot
-    const mattermost_site_url = call.context.mattermost_site_url;
+    const mattermostSiteURL = call.context.mattermost_site_url;
 
-    fetch(mattermost_site_url + '/api/v4/channels/direct', options).
-        then((mm_res) => mm_res.json()).
-        then((channel) => {
-            const post = {
-                channel_id: channel.id,
-                message,
-            };
+    const channel = await fetch(mattermostSiteURL + '/api/v4/channels/direct', options).
+        then((res) => res.json())
 
-            // Create a post
-            options.body = JSON.stringify(post);
+    const post = {
+        channel_id: channel.id,
+        message,
+    };
+
+    // Create a post
+    options.body = JSON.stringify(post);
+
+    fetch(mattermostSiteURL + '/api/v4/posts', options);
```

Comment:
We'll want to wait for this to be finished before calling `res.json`, otherwise we will run into the floating promises issue.

```suggestion
    await fetch(mattermostSiteURL + '/api/v4/posts', options);
```